styleguide/_themes/global/scss/_swatch.scss

```diff
@@ -3,6 +3,7 @@ $images: "https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.
 
 //Colors
 $white: #fff;
+$footer-gray: #b8b8b8;
```

Comment:
@WidgetsBurritos The colors in swatches are global brand approved colors we're allowed to use. Any addition to swatches to be super specific to derek should be put into a derek/_swatches.scss instead of global. I dont love having to define new colors for a hover effect in the first place but we def dont want to go cluttering up a global file with region and hover specific colors. @bighappyface what do you think?